src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -626,7 +633,9 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
 
         for (int i = 0; i < numOfRubricSubQuestions; i++) {
             StringBuilder tableBodyFragmentHtml = new StringBuilder();
+            StringBuilder tableBodyExcludingSelfFragmentHtml = new StringBuilder();
             boolean isSubQuestionRespondedTo = responseFrequency[i][numOfRubricChoices] > 0;
+            boolean isSubQuestionRespondedToForExcludingSelf = responseFrequencyExcludingSelf[i][numOfRubricChoices] > 0;
```

Comment:
`isSubQuestionRespondedToForExcludingSelf` can change to `isSubQuestionRespondedToExcludingSelf` to make it more consistent with how we identify excludingSelf variables